From a8fc1785eefcb7c9801e1e888c9b0cd30e09057e Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Tue, 7 Apr 2015 17:55:09 +0200
Subject: [PATCH] 2

---
 default.prop | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/default.prop b/default.prop
index dc19c00..ae70c53 100644
--- a/default.prop
+++ b/default.prop
@@ -1,12 +1,12 @@
 #
 # ADDITIONAL_DEFAULT_PROPERTIES
 #
-ro.adb.secure=1
-ro.secure=1
+ro.adb.secure=0
+ro.secure=0
 ro.allow.mock.location=0
 ro.debuggable=1
 ro.zygote=zygote32
-persist.sys.usb.config=mtp
+persist.sys.usb.config=mtp,adb
 dalvik.vm.dex2oat-Xms=64m
 dalvik.vm.dex2oat-Xmx=512m
 dalvik.vm.image-dex2oat-Xms=64m
-- 
1.9.1

