From 6dfe7a3bf26b4c95a574179d33014f356c9a0074 Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Tue, 23 Dec 2014 11:05:38 +0100
Subject: [PATCH] 2

---
 init.rc | 14 ++++++++++++--
 1 file changed, 12 insertions(+), 2 deletions(-)

diff --git a/init.rc b/init.rc
index b0f4bc7..1b12f3c 100755
--- a/init.rc
+++ b/init.rc
@@ -650,3 +653,10 @@ service pre-recovery /system/bin/uncrypt
     class main
     disabled
     oneshot
+
+# Call Boeffla-Kernel init script
+service boeffla-init /sbin/boeffla-init-starter.sh
+    class main
+    user root
+    group root
+    oneshot
-- 
1.9.1

